### PR TITLE
Update pointers to mixxx-devel mailing list

### PR DESCRIPTION
**Pull request by olafhering**

### CODE REVIEW

File: pages/support.html [hunk 1]
```diff
@@ -38,7 +38,7 @@ <h2>{% trans "Community Forums" %}</h2>
...
     <p>
-      {% blocktrans with mailing_list_link="https://lists.sourceforge.net/lists/listinfo/mixxx-devel" sourceforge_archive_link="http://sourceforge.net/mailarchive/forum.php?forum=mixxx-devel" gmane_archive_link="http://news.gmane.org/gmane.comp.multimedia.mixxx.devel" %}The <a href="{{ mailing_list_link}} ">mixxx-devel</a> developer mailing list (<a href="{{ sourceforge_archive_link }}">SourceForge</a> and <a href="{{ gmane_archive_link }}">Gmane</a> archives) is the primary means of developer communication.  The mailing list is where the Mixxx team discusses upcoming changes and programming related stuff.{% endblocktrans %}
+      {% blocktrans with mailing_list_link="https://lists.sourceforge.net/lists/listinfo/mixxx-devel" sourceforge_archive_link="https://sourceforge.net/p/mixxx/mailman/mixxx-devel/" %}The <a href="{{ mailing_list_link}} ">mixxx-devel</a> developer mailing list (<a href="{{ sourceforge_archive_link }}">SourceForge</a> and <a href="{{ gmane_archive_link }}">Gmane</a> archives) is the primary means of developer communication.  The mailing list is where the Mixxx team discusses upcoming changes and programming related stuff.{% endblocktrans %}
```
Comment (Be-ing) [inline pages/support.html:41]: looks like the gmane_archive_link was accidentally removed here

**Review by Be-ing — CHANGES_REQUESTED** [review]
Thanks for noticing this and opening this PR. It looks like you accidentally deleted a link though.

File: pages/support.html (hunk 1 above)
Comment (olafhering) [reply pages/support.html:41]: On purpose. That link is (or was) dead.

**Comment by rryan** [comment]
Thanks a lot for the change, but we removed the mailing list links.

**Pull request closed without merging**